### PR TITLE
Download audio files locally when rendering live 

### PR DESCRIPTION
**Pull request by championswimmer**
## SQUASH WHEN MERGING

### CODE REVIEW

**Comment by aayusharora** [comment]
@championswimmer Remove conflicts please.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "server.generator": "node src/generator/app.js",
     "start": "npm run server.generator",
     "test": "npm run server.generator&",
-    "tester":"./node_modules/.bin/jasmine-node test/spec"
+    "tester": "./node_modules/.bin/jasmine-node spec"
```
Comment (aayusharora) [inline package.json]: @championswimmer This should not be changed. It is required to run the test.

Comment (championswimmer) [inline package.json]: okay sorry

**Comment by aayusharora** [comment]
Able to download audio files. LGTM

**Pull request merged**